app/views/layouts/_footer.html.erb

```diff
@@ -17,16 +17,6 @@
             <li>Monday to Friday, 9am to 5:30pm<br>(except public holidays)</li>
           </ul>
         </div>
-
-        <div class="govuk-grid-column-one-third-from-desktop">
-          <h3 class="govuk-heading-s govuk-!-margin-bottom-1">Email</h3>
-          <ul class="govuk-list govuk-!-font-size-16 govuk-!-margin-bottom-0">
-            <li>
-              <%= govuk_mail_to("dluhc.digital-services@levellingup.gov.uk", subject: "CORE:", class: "govuk-footer__link") %>
-            </li>
-            <li class="govuk-!-margin-bottom-0">We aim to respond within 2 working&nbsp;days</li>
```

Comment:
Do we also want to remove the response time? Or move this to the helpdesk query section?

Reply:
Double checked this and we want to remove it!